appverifier/src/main/java/com/android/mdl/appreader/VerifierApp.kt

```diff
@@ -41,6 +50,29 @@ class VerifierApp : Application() {
         CredentialTypeRepository()
     }
 
+    /**
+     * Create a PresentationLogStore
+     */
+    private fun createPresentationLogStore(
```

Comment:
following the existing style of defining a `presentationLogStoreInstance` that need a context to be instantiated (and thus not a `lateinit var`)